_pages/ato.md

```diff
@@ -4,20 +4,32 @@ title: Lifecycle of a Launch
 
 Every federal information system must go through NIST's [Risk Management Framework](background/) before it can be used to process federal information. This process culminates in a signed Authority to Operate (ATO) being issued. Because the ATO process is a complex, multi-step process which will constrain the design and implementation of your system, you should start thinking about how it applies to your system _before_ you begin designing and implementing it.
 
-### Step 0 — Create ATO checklist
+### Timeline
+
+There are a few factors that will determine how long it takes a project to get an ATO. These map to [the checklist](checklist/), so might be helpful to open that up in another window and follow along.
+
+* Everything in [Phase 1](checklist/#phase-1-ato-sprint-prerequisites) needs to be done before the project can enter the ATO Sprint. That responsibility is mostly on the project team and the respective Infrastructure Lead. This could easily be 40 hours of work for a typical 18F project.
+* Projects get scheduled for [Phase 3](checklist/#phase-3-ato-sprint) based on order they are complete [Phase 2](checklist/#phase-2-documentation-review), and their priority (deadlines, etc).
```

Comment:
I think there's a typo here? Should it read "Projects get scheduled for Phase 3 based on *the order they complete* Phase 2... ?